buildSrc/subprojects/buildquality/src/main/kotlin/org/gradle/gradlebuild/buildquality/CiReportingPlugin.kt

```diff
@@ -36,24 +36,30 @@ open class CiReportingPlugin : Plugin<Project> {
 
     private
     fun Project.prepareReportsForCiPublishing() {
-        val failedTaskGenericHtmlReports = failedTasks(allprojects).flatMap {
+        val executedTaskGenericHtmlReports = executedTasks(allprojects).flatMap {
```

Comment:
I wonder if archiving the test result always will take too much disk space.

Reply:
Not at all AFAIK: https://builds.gradle.org/viewLog.html?buildId=31895112&buildTypeId=Gradle_Check_Quick_1_bucket50&tab=artifacts&branch_Gradle_Check_Quick_1=blindpirate%2Farchive-test-reports